packages/create-itp-react-app/README.md

```diff
@@ -1,34 +1,43 @@
 ## Generated project structure
-* docs
-* src
-  * app
-    * screens
-  * common (import from generated-project-name/common)
-    * assets
-    * components
-      * atoms
-      * molecules
-      * organisms
-    * primitives
-  * core (import from generated-project-name/core)
-    * actions
-    * lib
-    * reducers
-    * sagas
+
+- docs
+- src
+  - app
+    - screens
+  - common (import from generated-project-name/common)
+    - assets
+    - components
+      - atoms
+      - molecules
+      - organisms
+    - primitives
+  - core (import from generated-project-name/core)
+    - actions
+    - lib
+    - reducers
+    - sagas
 
 ### Local packages
-* generated-project-name/common
-* generated-project-name/core
+
+- generated-project-name/common
+- generated-project-name/core
+
+## Run the generator
+
+- run `./index <MY_PROJECT_NAME>`
+- a generated project is created in the folder <MY_PROJECT_NAME>
 
 ## docz
-* Added docz as a sub package because CRA uses webpack3, docz needs webpack4
-* Due to a docz bug, it is currently not possible to import external (as in outside the docz folder) components:
+
```

Comment:
no longer valid?

Reply:
we currently decided to use the catalog generator instead of docz, but idd, new documentation is required, I'll add some